shared/domain/src/main/java/gov/va/vro/model/mas/request/MasAutomatedClaimRequest.java

```diff
@@ -59,4 +60,15 @@ public class MasAutomatedClaimRequest {
 
   @Schema(description = "Veteran Flash Ids", example = "[\"999\"]")
   private List<String> veteranFlashIds;
+
+  public Boolean canProcess() {
+    List<String> validDisabilityActionTypes = Arrays.asList("NEW", "INCREASE");
+    if (!claimDetail.getConditions().getDiagnosticCode().equals("7101")) return false;
+    else if (!validDisabilityActionTypes.contains(
+        claimDetail.getConditions().getDisabilityActionType())) return false;
+    else if (claimDetail.getConditions().getDisabilityActionType().equals("NEW")
+        && (!veteranFlashIds.contains("Agent Orange Exposure Verified")
```

Comment:
This case will need to be processed seperate. For this case we will off-ramp as before; will not return 422.

Reply:
Fixed